src/bindings/python/setup.in.py

```diff
@@ -0,0 +1,48 @@
+""" libCellML Library: A library for the parsing, printing, and manipulation
+of CellML 2.0 compliant models.
+
+"""
+
+classifiers = """\
+Development Status :: 5 - Production/Stable
+Intended Audience :: Developers
+Intended Audience :: Education
+Intended Audience :: Science/Research
+License :: OSI Approved :: Apache Software License
+Programming Language :: Python
+Operating System :: Microsoft :: Windows
+Operating System :: Unix
+Operating System :: MacOS :: MacOS X
+Topic :: Software Development :: Libraries :: Python Modules
+"""
+
+from setuptools import setup
+from setuptools.dist import Distribution
+
+doclines = __doc__.split("\n")
+
+
+class BinaryDistribution(Distribution):
+    def is_pure(self):
+        return False
+
+    def has_ext_modules(self):
+        return True
+
+
+setup(
+    name='@PYPI_PACKAGE_NAME@',
+    version='@libCellML_VERSION@@LIBCELLML_DEVELOPER_VERSION@',
+    author='H. Sorby',
+    author_email='h.sorby@auckland.ac.nz',
```

Comment:
Wouldn't it be best to have a generic email address for libCellML? Anything that would be emailed to it would then be forwarded to some/all of us?

Reply:
Sure, is there one?

Reply:
I am not aware of one for the libCellML project, but that's maybe something we could look into (@nickerso?). However, this should clearly not block this PR.

Reply:
This will be resolved, if at all, as part of issue #336, so consider this conversation as resolved.